pam/pam_test/pam-module-transaction-dummy_test.go

```diff
@@ -0,0 +1,218 @@
+package pam_test
+
+import (
+	"testing"
+
+	"github.com/msteinert/pam"
+	"github.com/stretchr/testify/require"
+)
+
+func Test_ModuleTransactionDummy_SetItem(t *testing.T) {
+	t.Parallel()
+	tx := NewModuleTransactionDummy(nil)
+	err := tx.SetItem(pam.User, "an user")
+	require.NoError(t, err, "SetItem should not return an error")
+}
+
+func Test_ModuleTransactionDummy_GetItem_Empty(t *testing.T) {
+	t.Parallel()
+	tx := NewModuleTransactionDummy(nil)
+	val, err := tx.GetItem(pam.User)
+	require.NoError(t, err, "GetItem should not return an error")
+	require.Equal(t, "", val)
+}
+
+func Test_ModuleTransactionDummy_GetSetItem(t *testing.T) {
+	t.Parallel()
+	tx := NewModuleTransactionDummy(nil)
+	err := tx.SetItem(pam.User, "an user")
+	require.NoError(t, err, "SetItem should not return an error")
+	val, err := tx.GetItem(pam.User)
+	require.NoError(t, err, "GetItem should not return an error")
+	require.Equal(t, "an user", val)
+}
+
+func Test_ModuleTransactionDummy_GetEnv_Empty(t *testing.T) {
+	t.Parallel()
+	tx := NewModuleTransactionDummy(nil)
+	val := tx.GetEnv("AN_ENV")
+	require.Equal(t, "", val)
+}
+
+func Test_ModuleTransactionDummy_PutEnvGetEnv(t *testing.T) {
```

Comment:
Sure thing, I've used it a lot in the dependency (like https://github.com/3v1n0/go-pam/commit/a70d12f749d2015096c73b3cd8ce3777cdec9500 and many others commits I've in that branch and locally) but here I didn't find much gain in having it because at the end the common elements were not so many... So it would have saved me just saved me in having the sub-test name as string instead that as a function suffix...

So let me know if it's worth changing it here, but I'm not sure it will be :)